### PR TITLE
added wo topic

### PR DESCRIPTION
**Pull request by TordJoranger**

### CODE REVIEW

**Comment by TordJoranger** [comment]
Can bump version after checklist is added I think.

**Review by llykkelarsen — APPROVED** [review]

**Comment by paal123** [comment]
Remember to add the workorder topic  to servicebus dev, test & prod before completing this PR.

File: src/Equinor.ProCoSys.PcsBus/Topics/WorkOrderTopic.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        public string TypeOfWorkCode { get; set; }
+        public string OnShoreOffShoreCode { get; set; }
+        public string WoTypeCode { get; set; }
+        public string ProjectProgress { get; set; }
+        public string ExpendedManHours { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public DateTime LastUpdated { get; set; }
+
+        public const string TopicName = "workorder";
+
```
Comment (paal123) [inline src/Equinor.ProCoSys.PcsBus/Topics/WorkOrderTopic.cs:40]: Empty line

File: src/Equinor.ProCoSys.PcsBus/Topics/WorkOrderTopic.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Equinor.ProCoSys.PcsServiceBus.Topics
+{
+    public class WorkOrderTopic
+    {
+
```
Comment (paal123) [inline src/Equinor.ProCoSys.PcsBus/Topics/WorkOrderTopic.cs:11]: Empty line

**Review by paal123 — APPROVED** [review]
With suggestions

**Comment by TordJoranger** [comment]
AB#84484

**Pull request merged**